vent/menu.py

```diff
@@ -6,6 +6,9 @@
 import npyscreen
 import subprocess
 
+from helpers.meta import Version
```

Comment:
@cglewis Just tested by removing my `vent` packages and changing to `vent.helpers.meta` and it broke: `ImportError: No module named vent.helpers.meta`